docs/api/programme.md

```diff
@@ -0,0 +1,253 @@
+# Programme
+
+A programme represents a type of graduation such as LEIC or MEIC. 
+A programme is composed by a set of _offers_. 
+
+## Properties
+   * Name
+      -  mandatory: the programme's name
+      -  type: text
+      -  e.g. "Licenciatura em Engenharia Informática e de Computadores"
+    
+   * Acronym: programme's acronym
+      - mandatory
+      - type: text
+      - e.g. "LEIC"
+
+   * Term Size: Duration in terms
+      - mandatory
+      - type: number
+      - e.g. 6 terms for LEIC, 4 terms for MEIC
+
+
+## Link Relations
+   * A programme representation
+      - must include a link to its context, using the self link relation.
+      - may include a link to the collection it belongs, using the collection link relation.
+      - must include a link to its _offers_.
+
+## Actions
+    * edit: edit's a programme
+    * add Offer: add's a curricular unit offer to the programme
+
+## Example representation
+```json
+{
+    "class": [ "programme" ],
+    "properties": {
+        "name": "Licenciatura em Engenharia Informática e de Computadores",
+        "acronym": "LEIC",
+        "termSize": 4
+    },
+    "entities": [
+        {
+            "class": [ "offer" ],
+            "title": "LS Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "links" : [
+                { "rel": [ "self" ], "href": "/v0/programmes/LEIC/offers/1"}
+            ]
+        },
+        {
+            "class": [ "offer" ],
+            "title": "AED Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "links" : [
+                { "rel": [ "self" ], "href": "/v0/programmes/LEIC/offers/2"}
+            ]
+        },
+        {
+            "class": [ "offer" ],
+            "title": "POO Offer",
+            "rel": [ "/rel/programmeOffer" ],
+            "links" : [
+                { "rel": [ "self" ], "href": "/v0/programmes/LEIC/offers/3"}
+            ]
+        }
+    ],
```

Comment:
Objects in the `entities` property can have a `properties` property. It could be interesting to have the terms in which the course is offered